modules/commit-parser.js

```diff
@@ -25,7 +25,7 @@ const parseCommitMessage = (commitMsg, commitHash, settings) => {
 
   result.body = body
   result.type = type || ""
-  result.release = breaking === "!" ? "major" : types[type]?.release || "none"
+  result.release = breaking === "!" ? "major" : types[type]?.release || "patch"
```

Comment:
that could have unintended consequence. Could you refactor the PR so you only change what it is absolutely necessary? That would help me a lot to effectively review and ensure no breaking changes are introduced.